src/React.re

```diff
@@ -24,6 +24,18 @@ external createElementVariadic:
   (component('props), 'props, array(element)) => element =
   "createElement";
 
+[@bs.module "react"] [@ocaml.deprecated "Please use JSX syntax directly."]
+external jsxKeyed: (component('props), 'props, string) => element = "jsx";
+
+[@bs.module "react"] [@ocaml.deprecated "Please use JSX syntax directly."]
+external jsx: (component('props), 'props) => element = "jsx";
+
+[@bs.module "react"] [@ocaml.deprecated "Please use JSX syntax directly."]
+external jsxs: (component('props), 'props) => element = "jsxs";
+
+[@bs.module "react"] [@ocaml.deprecated "Please use JSX syntax directly."]
+external jsxsKeyed: (component('props), 'props, string) => element = "jsxs";
+
```

Comment:
The idea here is to have the ppx disable these warnings. Wish we were working with OCaml 4.08 - https://github.com/ocaml/ocaml/pull/1804